plugins/source/azuredevops/codegen/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"log"
+
+	"github.com/cloudquery/cloudquery/plugins/source/azuredevops/codegen/recipes"
+)
+
+func main() {
+	var resources []*recipes.Resource
+	resources = append(resources, recipes.Core()...)
```

Comment:
I'm not sure, but maybe we could do `All()` func in `recipes` package that will do append on its own?

Reply:
We're not very consistent with this pattern across plugins, AWS uses append, Azure uses static `init()` with a `var`.
I'll follow the Azure plugin as that's the newest version of code gen we have.